cypress/integration/bulk-edit/csv/bulk-edit-csv-usernames.spec.js

```diff
@@ -0,0 +1,63 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import testTypes from '../../../support/dictionary/testTypes';
+import permissions from '../../../support/dictionary/permissions';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import devTeams from '../../../support/dictionary/devTeams';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import Users from '../../../support/fragments/users/users';
+
+let user;
+const userUUIDsFileName = `userUUIDs_${getRandomPostfix()}.csv`;
+const matchRecordsFileName = `matchedRecords_${getRandomPostfix()}.csv`;
+const importFileName = `bulkEditImport_${getRandomPostfix()}.csv`;
+
+describe('bulk-edit', () => {
+  describe('csv approach', () => {
```

Comment:
Is it convenient in this case to use two describes? you can just name the first decribe like ```bulk-edit: csv approach```

Reply:
This is for allure report. I want picture like with grouping like this:

![image](https://user-images.githubusercontent.com/26041116/183408803-a441b147-1710-4f6f-b7e7-442bdd0941b5.png)

If I will use one describe I get this picture:
![image](https://user-images.githubusercontent.com/26041116/183408954-a038bbd4-d291-4e3a-aaf8-610a08da7377.png)

Reply:
Got it.